iac-extensions/arm/src/test/resources/checks/HardCodedCredentialsCheck/HardCodedCredentialsCheck.bicep

```diff
@@ -0,0 +1,24 @@
+resource sqlServer 'Microsoft.Sql/servers@2023-05-01-preview' = {
+  name: 'noncompliant'
+  properties: {
+    administratorLogin: 'admin' // Noncompliant {{Revoke and change this secret, as it might be compromised.}}
+//  ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    administratorLoginPassword: 'password' // Noncompliant
+    password: 'password' // Noncompliant
+    secret: 'password' // Noncompliant
+    adminPassword: 'password' // Noncompliant
+    adminUsername: 'admin' // Noncompliant
+  }
+}
+
+resource sqlServer 'Microsoft.Sql/servers@2023-05-01-preview' = {
+  name: 'compliant'
+  properties: {
+    administratorLogin: adminLogin
+    administratorLoginPassword: adminPassword
+    password: password
+    secret: secret
+    adminPassword: adminPassword
+    adminUsername: ''
```

Comment:
please also add a case when the value contains only spaces - I refer to `! String.isBlank()` condition in implementation.